### DISCUSSION TITLE
HOW TO RUN THIS

### DISCUSSION THREAD

**Discussion by ashrit67** [AnveshakR/poster-generator#16] [Q&A] [answered]
HI I WANT TO RUN THE WEB APP BUT I AM GETTING A ACCESS_TOKEN ERROR
![image](https://github.com/user-attachments/assets/e487bce8-a606-44dd-9a3f-68c44647d885)

**Comment by AnveshakR** [comment, marked as answer]
Hello @ashrit67 , did you create the `.env ` file with the Spotify API credentials as detailed in the README? That error only pops up if the Spotify API returns a failure due to invalid/missing credentials.
